tgui/packages/tgui/styles/components/Button.scss

```diff
@@ -1,188 +0,0 @@
-/**
- * Copyright (c) 2020 Aleksej Komarov
- * SPDX-License-Identifier: MIT
- * Changes: ZeWaka (FA6)
- * Changes: Mordent (FA6)
- */
-
-@use '../base';
-@use '../colors';
-@use '../functions' as *;
-
-$color-default: colors.bg(colors.$primary) !default;
-$color-disabled: hsl(0, 0%, 60%) !default;
-$color-selected: colors.bg(colors.$green) !default;
-$color-caution: colors.bg(colors.$yellow) !default;
-$color-danger: colors.bg(colors.$red) !default;
-$color-transparent-text: rgba(255, 255, 255, 0.5) !default;
-$border-radius: base.$border-radius !default;
-$bg-map: colors.$bg-map !default;
-
-@mixin button-color($color) {
-  // Adapt text color to background luminance to ensure high contast
-  $luminance: luminance($color);
-  $text-color: if($luminance > 0.4, rgba(0, 0, 0, 1), rgba(255, 255, 255, 1));
-
-  transition:
-    color 50ms,
-    background-color 50ms;
-  background-color: $color;
-  color: $text-color;
-
-  &:focus {
-    transition:
-      color 100ms,
-      background-color 100ms;
-  }
-
-  &:hover {
-    background-color: lighten($color, 30%);
-    color: $text-color;
-  }
-}
-
-.Button {
-  position: relative;
-  display: inline-block;
-  line-height: 1.667em;
-  padding: 0 0.5em;
-  margin-right: base.em(2px);
-  white-space: nowrap;
-  outline: 0;
-  border-radius: $border-radius;
-  margin-bottom: base.em(2px);
-  // Disable selection in buttons
-  user-select: none;
-  -ms-user-select: none;
-
-  &:last-child {
-    margin-right: 0;
-    margin-bottom: 0;
-  }
-
-  .fa,
-  .fa-solid, // |GOONSTATION-CHANGE|
-  .fa-regular // |GOONSTATION-CHANGE|
-  {
-    margin-left: 0; // ZeWaka: Changed from -0.25em due to FA6 offset
-    margin-right: 0; // Mordent: Changed from -0.25em due to FA6 offset
-    min-width: 1.333em;
-    text-align: center;
-  }
-}
-
-.Button--dropdown {
-  line-height: base.em(16px);
-  height: base.em(22px);
-  padding: 0.2rem 0.5rem;
-}
-
-.Button--hasContent {
-  // Add a margin to the icon to keep it separate from the text
-  .fa,
-  .fa-solid, // |GOONSTATION-CHANGE|
-  .fa-regular // |GOONSTATION-CHANGE|
-  {
-    margin-right: 0.25em;
-  }
-}
-
-.Button--hasContent.Button--iconPosition--right {
-  .fa,
-  .fa-solid, // |GOONSTATION-CHANGE|
-  .fa-regular // |GOONSTATION-CHANGE|
-  {
-    margin-right: 0px;
-    margin-left: 3px;
-  }
-}
-
-.Button--ellipsis {
-  display: block;
-  text-overflow: ellipsis;
-  overflow: hidden;
-}
-
-// |GOONSTATION-ADD|
-// ellipsis has to be on the children, it can't work otherwise.
-.Button--ellipsis .Button__content {
-  overflow: hidden;
-  text-overflow: ellipsis;
-}
```

Comment:
Check upstream.

Reply:
Shouldn't be needed anymore as they fixed it in https://github.com/tgstation/tgstation/pull/80194

Reply:
Verified that change got through to tgui-styles, seems fine.